da/mock/mock.go

```diff
@@ -2,11 +2,13 @@ package mock
 
 import (
 	"encoding/binary"
+	"errors"
 
 	"github.com/celestiaorg/optimint/da"
 	"github.com/celestiaorg/optimint/log"
 	"github.com/celestiaorg/optimint/store"
 	"github.com/celestiaorg/optimint/types"
+	"github.com/dgraph-io/badger"
```

Comment:
This will be removed after changing `badger.KeyNotFound` to `store.KeyNotFound`.